src/Symfony/Component/Validator/Resources/translations/validators.zh_CN.xlf

```diff
@@ -310,10 +310,30 @@
                 <source>This value does not match the expected {{ charset }} charset.</source>
                 <target>该值不符合 {{ charset }} 编码。</target>
             </trans-unit>
+            <trans-unit id="81">
+                <source>This is not a valid Business Identifier Code (BIC).</source>
+                <target>这不是有效的业务标识符代码（BIC）。</target>
```

Comment:
Not sure, but it looks like there is an extra space before the dot.

Reply:
that space is part of `U+FF09 fullwidth right parenthesis`

https://en.wikipedia.org/wiki/Chinese_punctuation#Marks_similar_to_European_punctuation